.gitignore

```diff
@@ -30,3 +30,4 @@ sync-local.sh
 
 Userland/Libraries/LibWasm/Tests/Fixtures/SpecTests
 Userland/Libraries/LibWasm/Tests/Spec
+Base/home/anon/Games/halflife
```

Comment:
Please don't do this. `You can put custom stuff in Build/$(SERENITY_ARCH)/Root/home/anon/Games/...`
No need to pollute `Base/` which then needs `.gitignore` hax :)

Reply:
will do, captain!